build/vendir.yml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: vendir.k14s.io/v1alpha1
+kind: Config
+minimumRequiredVersion: 0.11.0
+directories:
+- path: sources/cf-k8s-logging-fluent
+  contents:
+  - path: .
+    git:
+      url: https://github.com/cloudfoundry/cf-k8s-logging-fluent.git
+      ref: 35b578f86027750372852635a3a291f4caf04e09
```

Comment:
I'm still unsure if this vendir.yml file is really needed. Even though cf-k8s-logging pulls in this fluent repo, it is not actually referenced in the `generated-kbld-config.sh`

After reviewing it further, even in the `kbld.yml` file references it from `68: sources/logging/vendor/cf-k8s-logging-fluent/fluentd`